enhancements/image-references.md

```diff
@@ -0,0 +1,228 @@
+---
+title: image-references-in-operator-bundles
+authors:
+  - "@stevekuznetsov"
+reviewers:
+  - "@jwforres"
+  - "@shawn-hurley"
+  - "@gallettilance"
+  - "@lui"
+approvers:
+  - "@jwforres"
+  - "@shawn-hurley"
+creation-date: 2020-05-17
+last-updated: 2020-05-17
+status: implementable
+see-also:
+  - "/enhancements/olm/operator-bundle.md"
+---
+
+# Image References in Operator Bundle Manifests
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Manifests that make up an Operator Bundle for installation via the Operator
+Lifecycle Manager refer to one or more container images by their pull
+specifications: these container images define the operator and operands that
```

Comment:
Isn't the operand only referred to by resources the operator creates?

Reply:
Not sure i understand this question.

Reply:
Any images required by the operator (for operands, etc) need to be surfaced at the packaging layer for mirroring to disconnected environments.